packages/react-dom/src/useInView.spec.tsx

```diff
@@ -104,16 +104,16 @@ it('should trigger onChange', () => {
   render(<HookComponent options={{ onChange }} />)
 
   mockAllIsIntersecting(true)
-  expect(onChange).toHaveBeenLastCalledWith(
-    true,
-    expect.objectContaining({ intersectionRatio: 1, isIntersecting: true })
-  )
+  expect(onChange).toHaveBeenLastCalledWith({
+    isInView: true,
+    entry: expect.objectContaining({ intersectionRatio: 1, isIntersecting: true }),
+  })
```

Comment:
This is changed too @kangju2000 @gwansikk Check it out please

Reply:
Makes sense to me. No strong opinion either way.